Formula/openssl@3.rb

```diff
@@ -108,6 +110,9 @@ def openssldir
   def post_install
     rm_f openssldir/"cert.pem"
     openssldir.install_symlink Formula["ca-certificates"].pkgetc/"cert.pem"
+    system "#{bin}/openssl", "fipsinstall",
+           "-out", "#{openssldir}/fipsmodule.cnf",
+           "-module", "#{lib}/ossl-modules/fips.dylib"
```

Comment:
`make install` should be doing this automatically. Can you explain why this manual step is required?

Reply:
See at https://github.com/openssl/openssl/blob/master/README-FIPS.md#installing-the-fips-module, which says

> The FIPS module must have the self tests run, and the FIPS module config file output generated on every machine that it is to be used on. You must not copy the FIPS module config file output data from one machine to another.

Reply:
Ah yes, thanks - I missed that.

How long does this step take to run?

Reply:
> How long does this step take to run?

On the order of 0.050 seconds for me.

Reply:
Oh ok that sounds good. Self tests sounded like something that would've taken longer.